resources/incubator/images/spindle/network.svg

```diff
@@ -0,0 +1,613 @@
+         sodipodi:role="line"
+         x="111.56572"
+         y="70.506142"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52777px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264999;stroke-miterlimit:4;stroke-dasharray:none"
+         id="tspan38881-2">when model is trained,</tspan><tspan
+         sodipodi:role="line"
+         x="111.56572"
+         y="74.921173"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52777px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264999;stroke-miterlimit:4;stroke-dasharray:none"
+         id="tspan39884">get prediction</tspan></text>
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-weight:normal;font-size:10.5833px;line-height:1.25;font-family:sans-serif;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.264583"
+       x="20.587292"
+       y="87.922607"
+       id="text39274"><tspan
+         sodipodi:role="line"
+         id="tspan39272"
+         x="20.587292"
+         y="87.922607"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52778px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264583" /></text>
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-weight:normal;font-size:10.5833px;line-height:0.35;font-family:sans-serif;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.264583"
+       x="85.337532"
+       y="170.93884"
+       id="text39814"><tspan
+         sodipodi:role="line"
+         id="tspan39812"
+         x="85.337532"
+         y="170.93884"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52778px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264583">every network round, merge</tspan><tspan
+         sodipodi:role="line"
+         x="85.337532"
+         y="175.35387"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52778px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264583"
+         id="tspan39816">the hospital local's models</tspan></text>
```

Comment:
```suggestion
         id="tspan39816">the hospital local models</tspan></text>
```
It is not clear to me where this happens. Do all the hospitals, at every network round, merge the local models of all the other hospitals into their own local model?